torch/_inductor/ir.py

```diff
@@ -7381,11 +7381,11 @@ def create(cls, collective_op: "TensorBox"):
 
     def get_inputs_that_alias_output(self):
         # Signal to codegen that our output buffer isn't safe to reuse
-        return [self.inputs[0].codegen_reference()]
+        return [self.inputs[0].get_name()]
```

Comment:
This is deprecated code. But curious if you know if there was any significance of using `codegen_reference`.

Reply:
No. Pretty sure it never was right. See https://github.com/pytorch/pytorch/pull/108811#discussion_r1319338204